handbook/customers/README.md

```diff
@@ -487,6 +487,43 @@ In order to maintain a consistent contributor experience in Salesforce, we log i
 * Default filter should be all leads. Open a lead to continue.
 * There should be fields for name, lead source, lead status, and rating.
 
+### Solutions consulting
+What is it like to be a Solutions Consultant at Fleet? What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? 
+
+#### Within 30 days -- assess and absorb information
+* Being new to the company, you have fresh eyes. Document everything that seems odd to you (process, product). 
+* Begin working on domain and product knowledge acquisition. Complete guides found at https://fleetdm.com/guides.
+* Be a prospect. Sign up for a sandbox trial. What makes sense? What doesn't? Document. 
+* Understand Fleet value drivers. Why do Fleet Community users want to give us money to upgrade to Premium? 
+* Complete consensus training videos (see the "Description" field in the `#help-consensus-tool` slack channel.
+* Review [gong](gong.io) calls.
+* Get trained on [Demo2Win](https://2winglobal.com/about-you/pre-sales/)
+
+#### First 60 days -- Synthesize
```

Comment:
remove capitalization on synthesize for consistency

Reply:
yup, I saw that right when I was making the PR ;(